README.rst

```diff
@@ -0,0 +1,81 @@
+SaltyRTC Signalling Server
+==========================
+
+|Travis| |codecov| |PyPI|
+
+This is a SaltyRTC server implementation for Python 3.4+ using
+`asyncio`_.
+
+Note
+****
+
+On machines where Python 3 is not the default Python runtime, you should
+use ``pip3`` instead of ``pip``.
+
+Prerequisites
+*************
+
+.. code-block:: bash
+
+    $ sudo apt-get install python3 python3-pip
+
+We recommend using `venv`_ to create an isolated Python environment:
+
+.. code-block:: bash
+
+    $ pyvenv venv
+
+You can switch into the created virtual environment *venv* by running
+this command:
+
+.. code-block:: bash
+
+    $ source venv/bin/activate
+
+While the virtual environment is active, all packages installed using
+``pip`` will be installed into this environment.
+
+To deactivate the virtual environment, just run:
+
+.. code-block:: bash
+
+    $ deactivate
+
+If you want easier handling of your virtualenvs, you might also want to
+take a look at `virtualenvwrapper`_.
+
+Installation
+************
+
+If you are using a virtual environment, activate it first.
+
+Install the module by running:
+
+.. code-block:: bash
+
+    $ pip install saltyrtc
```

Comment:
Hm, but a client will probably require dependencies like WebRTC. And I don't think it makes sense to install client code on a server. The server should only have access to as little code as necessary. It would also mean that a client-only code update would bump the version number for the server, requiring an update.

I'd definitely prefer the first approach, where the server and client packages are kept small, and common code may be extracted into a "saltyrtc-common" or "saltyrtc-utils" package that is a dependency of both.